audio/reference/reference_descriptors

```diff
@@ -1,783 +0,0 @@
-Bus 001 Device 058: ID 046d:082b Logitech, Inc. Webcam C170
```

Comment:
Did you remove these files deliberately?

Reply:
Yes, as they are not much use past the initial setup of the descriptors